src/graphql/search/cms/normalize.ts

```diff
@@ -89,16 +90,17 @@ function getFormattedResults(results: any): Array<CmsSearchResultType> {
 
 function getThemeFilterOptions(
   result: JsonAPI,
-  themeCount: Array<{ key: string; count: number }>,
+  themeCount?: Array<{ key: string; count: number }>,
 ): Array<FilterOptions> {
-  return result.data.map((item: any) => {
-    const id = item.attributes.drupal_internal__tid
+  return result.data.map(({ attributes }: any) => {
+    const id = attributes.drupal_internal__tid
+    const { count } = themeCount?.find(count => count.key === id) || {}
 
-    const { count } = themeCount.find(count => count.key === id) || {}
+    const key = Object.keys(DrupalThemeFilterIDs).find(x => DrupalThemeFilterIDs[x] === id) || ''
```

Comment:
Calling `Object.keys` here will return all values and keys of the enum.

For example:
```ts
enum Test {
  Foo = 1,
  Bar = 2,
  Baz = 3
}

console.log(Object.keys(Test)) // ["1", "2", "3", "Foo", "Bar", "Baz"]
```

This might have some implications for this implementation.